utils/python/CIME/XML/GenericXML.py

```diff
@@ -31,7 +31,7 @@ def __init__(self, infile=None):
             self.filename = infile
             self.root = ET.Element(infile)
             self.root.set('version','1.0')
-            
```

Comment:
Remove trailing whitespace. I have my emacs set to highlight trailing whitespace, so I notice it right away. It's a bit of a nitpick, but it can result in problems.
http://programmers.stackexchange.com/questions/121555/why-is-trailing-whitespace-a-big-deal